collectorist/api/src/db/mod.rs

```diff
@@ -329,4 +384,47 @@ mod test {
 
         Ok(())
     }
+
+    #[actix_web::test]
```

Comment:
I don't know this thing but if you added then I'm "ok"

Reply:
It's used to enable async tests. #[tokio::test] is also viable.